patterns/2-structured/maturity-model.md

```diff
@@ -34,7 +34,7 @@ Teams sharing InnerSource learnings run into misunderstandings as they are not
 aware of their respective level of InnerSource adoption.
 
 Teams believe that "it's all about migrating to a shared software development
-forge".
+[forge](https://en.wikipedia.org/wiki/Forge_(software))" (GitLab, GitHub, or Bitbucket being well known examples of such forges).
```

Comment:
```suggestion
[forge](https://en.wikipedia.org/wiki/Forge_%28software%29)" (GitLab, GitHub, or Bitbucket being well known examples of such forges).
```

Reply:
Changing the encoding of the parenthesis will hopefully allow the automatic Link Check to work correctly.